components/tools/OmeroWeb/omeroweb/api/views.py

```diff
@@ -251,12 +312,28 @@ def get_opts(self, request, **kwargs):
                 opts['project'] = project
         return opts
 
+    # Urls to add to marshalled object. See ProjectsView for more details
+    urls = {
+        'dataset_url': {'name': 'api_dataset',
```

Comment:
maybe keep the same order for the url list in each view
i.e. retrieval of children then object e.g. datasets/project